### PR TITLE
test: workaround for cypress-tags and other small fixes

### PR DESCRIPTION
**Pull request by nalmabrcom**
`cypress-tags` temporarily is not working (check this [issue](https://github.com/annaet/cypress-tags/issues/58) ). So I've decided to comment those tests are not working (have tags `flaky,flaky_theia,bug,investigation`). So `CYPRESS_EXCLUDE_TAGS` or `CYPRESS_INCLUDE_TAGS` are not working. 

```
====================================================================================================

  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  LSP/copybook.spec.js                     03:17       22       22        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  LSP/F101836.spec.js                      00:10        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  LSP/F102470.spec.js                      01:38        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  LSP/F95833.spec.js                       04:44        4        2        2        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  LSP/F95834.spec.js                       00:10        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  LSP/F96117.spec.js                       02:15        7        7        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  LSP/F96125.spec.js                       01:05        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  LSP/F97476.spec.js                       05:01        8        7        1        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  LSP/lsp.spec.js                          11:17       22       19        3        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  LSP/zowe.spec.js                         06:02       10       10        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✖  3 of 10 failed (30%)                     35:43       83       77        6        -        - 
 
 ```
Signed-off-by: Maryna Nalbandian <maryna.nalbandian@broadcom.com>

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
🥥

File: tests/cypress/integration/LSP/F95833.spec.js [hunk 1]
```diff
@@ -50,21 +50,21 @@ context('This is F95833 spec', () => {
...
-      cy.openFile('ADSORT.cbl').goToLine(59);
-      cy.getLineByNumber(59).findText('XCTL1').goToDefinition();
-      cy.getCurrentLineNumber().should('eq', 28);
-      cy.getCurrentLine().contains('XCTL1');
-      cy.getLineByNumber(59).type('{end}{enter}');
-      cy.getLineByNumber(60).type('EXEC CICS XCTL RESP(XCTL1) PROGRAM (XCTL1) END-EXEC.');
-      cy.pause();
-      cy.getLineByNumber(60).findText('XCTL1').goToDefinition();
-      cy.getCurrentLineNumber().should('eq', 28);
+  // describe('TC312738 CICS variables and paragraphs support', () => {
```
Comment (grianbrcom) [inline tests/cypress/integration/LSP/F95833.spec.js:53]: Is it makes sense to use `.skip` here instead?

File: tests/cypress/integration/LSP/copybook.spec.js [hunk 2]
```diff
@@ -316,21 +316,21 @@ context('This is a Copybook spec', () => {
     });
   });
 
-  describe.skip('TC312878 Edit missing copybook and resolve', () => {
```
Comment (grianbrcom) [inline tests/cypress/integration/LSP/copybook.spec.js:319]: Looks like I did a wrong assumption about the `skip` ability.

**Review by grianbrcom — APPROVED** [review]
:label:

**Pull request merged**